docs/guide/config/preferences.md

```diff
@@ -5,9 +5,15 @@ Xournal ++ has multiple options that can be configured and customized. Most of t
 
 ## Load / Save
 
-### Autosaving 
+### Autosaving
 
-Enable or disable automatic saving and set the time interval in which it will be done. If the document was previously saved in any folder, the autosave document will be in the same location as a hidden file. Otherwise the file will be saved in the path **~/.xournalpp/autosave**.
+Enable or disable automatic saving and set the time interval in which it will be done. If the document was previously saved in any folder, the autosave document will be in the same location as a hidden file. Otherwise the file will be saved in **$XDG_CONFIG_HOME/xournalpp/autosave** or if XDG_CONFIG_HOME is not set in **~/.config/xournalpp/autosave**. TODO: Windows
```

Comment:
Instead of explicitly listing the location for each platform, I think it would be better to refer the reader to the "Config folder" listed in [File Locations](https://xournalpp.github.io/guide/file-locations/).